From 1a7122b84fbb8e8be60316a5377d831ff66ff2cb Mon Sep 17 00:00:00 2001
From: John Crispin <john@phrozen.org>
Date: Sun, 16 Jul 2023 17:30:49 +0200
Subject: [PATCH 23/55] base-files: drop nand.sh from base-files

Removes NAND upgrade script from ipq95xx and ipq53xx targets where it conflicts with custom upgrade procedures.

Signed-off-by: John Crispin <john@phrozen.org>
---
 package/base-files/Makefile | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/package/base-files/Makefile b/package/base-files/Makefile
index b5f1d44ae1..4532d99e0e 100644
--- a/package/base-files/Makefile
+++ b/package/base-files/Makefile
@@ -153,6 +153,12 @@ ifeq ($(CONFIG_NAND_SUPPORT),)
   endef
 endif
 
+ifeq ($(CONFIG_TARGET_ipq95xx)$(CONFIG_TARGET_ipq53xx),y)
+  define Package/base-files/drop-nand-sh
+	rm -f $(1)/lib/upgrade/nand.sh
+  endef
+endif
+
 ifeq ($(CONFIG_EMMC_SUPPORT),)
   define Package/base-files/emmc-support
 	rm -f $(1)/lib/upgrade/emmc.sh
@@ -170,6 +176,7 @@ define Package/base-files/install
 	$(CP) ./files/* $(1)/
 	$(Package/base-files/install-key)
 	$(Package/base-files/nand-support)
+	$(Package/base-files/drop-nand-sh)
 	$(Package/base-files/legacy-sdcard-support)
 	$(Package/base-files/emmc-support)
 	if [ -d $(GENERIC_PLATFORM_DIR)/base-files/. ]; then \
-- 
2.34.1

